src/frontend/components/UI/PopoverComponent/index.scss

```diff
@@ -0,0 +1,12 @@
+.MuiPaper-root.MuiPopover-paper {
+  background-color: var(--background);
+  color: var(--text-primary);
+  border: 1px solid var(--border-color);
+  border-radius: 4px;
+  box-shadow: 0 0 0 1px var(--border-color);
+  padding: 16px;
+  width: fit-content;
+  max-width: 500px;
+  overflow-y: auto;
+  overflow-x: hidden;
+}
```

Comment:
This CSS is affecting the context menus too, it has extra padding now for example:
![image](https://user-images.githubusercontent.com/188464/219909204-457ade81-4415-467d-91e7-621df231cee0.png)